helm/examples/nfs/README.md

```diff
@@ -23,3 +24,5 @@ NB: Either use the output auto-generated when installing the "hpcc-nfs" helm cha
 ### Notes
 
 The main use case for this example is with GKE (Google Kubernetes Engine).
+
+"forcePermissions: true" is required for each storage. Also only one device for a data plane (numDevices should be 1 if defined) is supported
```

Comment:
...is required for each storage plane. Only supports one device per data plane (If defined numDevices should be 1).

Reply:
Updated

Reply:
Verified again